assets/sass/modules/_consent.scss

```diff
@@ -0,0 +1,162 @@
+.consent-header {
+  background-color: #f7f7f7;
```

Comment:
Note: didn't add the new colors to variables since this UI will be changed to match more the signin widget style